src/main/resources/application-dev.yml

```
@@ -36,4 +36,37 @@ spring:
+              - profile_image
+        provider:
+          kakao:
+            authorization-uri: https://kauth.kakao.com/oauth/authorize
+            token-uri: https://kauth.kakao.com/oauth/token
+            user-info-uri: https://kapi.kakao.com/v2/user/me
+            user-name-attribute: id
+
+---
+jwt:
+  secret: ${JWT_SECRET}
+  access-expiration: 2592000000 # 30일
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)